### DISCUSSION TITLE
No Results Returned

### DISCUSSION THREAD

**Discussion by jamessdixon** [vdemichev/DiaNN#307] [General]
I am using this mzML file: https://github.com/ryanlayerlab/hypedsearch/blob/main/data/spectra/hybrid_nod2e3.mzML
and this database file:
https://github.com/ryanlayerlab/hypedsearch/blob/main/data/database/filtered_database.fasta
and I am not getting any report file generated. 
Am I missing a step?

![image](https://user-images.githubusercontent.com/1394644/153775653-471ed200-9465-462c-96ba-abb6548c990f.png)

**Comment by vdemichev** [comment]
Something wrong with that .mzML file. Is it a valid DIA file obtained using MSConvert using the settings recommended in DIA-NN docs?

Vadim

**Comment by jamessdixon** [comment]
I don't know how that MzML file was generated.  Do you have a sample MzML file that works I can use to test and then compare?
